### PR TITLE
test(storybook): add cypress tests for storybook

### PR DESCRIPTION
**Pull request by David-Cohen35**
testing out a fix by adding packages with yarn instead of npm

### CODE REVIEW

**Review by reywright — CHANGES_REQUESTED** [review]
Rejecting per our conversation (feel free to dismiss my review when Cypress tests, sans component testing are ready for review)

**Comment by David-Cohen35** [comment]
will want to discuss folder structure - ideally to have a similar structure to jest which uses testRegex

File: package.json [hunk 1]
```diff
@@ -91,6 +93,7 @@
     "@storybook/components": "6.4.9",
     "@storybook/core-events": "6.4.9",
     "@storybook/react": "6.4.9",
+    "@storybook/testing-react": "^1.2.2",
```
Comment (reywright) [inline package.json:96]: This can be removed

File: package.json [hunk 2]
```diff
@@ -219,6 +223,7 @@
   "dependencies": {
     "@popperjs/core": "2.11.0",
     "@reach/menu-button": "0.16.2",
+    "@storybook/client-api": "6.4.10",
```
Comment (reywright) [inline package.json:226]: * Is this needed?
* If so, is it needed as a dependency?

File: cypress/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress"],
+    "jsx": "preserve"
+  },
+  "include": ["**/*.ts", "storybook/storybook.spec.ts"]
+}
```
Comment (reywright) [inline cypress/tsconfig.json:9]: Is this tsconfig file needed?

File: cypress/storybook/storybook.spec.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable jest/expect-expect */
```
Comment (reywright) [inline cypress/storybook/storybook.spec.ts:1]: sure... but if we're setting up our jest regex to exclude these files, then our linter should also exclude these files ideally.

**Review by reywright — COMMENTED** [review]
Some comments and questions.

**Review by ivanhdez28 — COMMENTED** [review]
other than what @reywright pointed out, this looks good to me, though i'm not familiar at all with cypress so i'd merge only after he approves it.

File: package.json (hunk 2 above)
Comment (David-Cohen35) [reply package.json:226]: not needed

File: cypress/tsconfig.json (hunk 3 above)
Comment (David-Cohen35) [reply cypress/tsconfig.json:9]: turns out i do not

File: cypress/storybook/storybook.spec.ts (hunk 4 above)
Comment (David-Cohen35) [reply cypress/storybook/storybook.spec.ts:1]: yeah meant to remove that - am still unsure why eslint is still saying that the test has no assertions - but it runs and passes either way

**Review by reywright — APPROVED** [review]
LGTM!

**Pull request merged**